client/src/selectors/profileSelectors.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import {createSelector} from 'reselect'
+import {sortBy, last, mapValues} from 'lodash'
+import {normalizeName, parseQueryFromLocation} from '../utils'
+import {paramsIdSelector} from './index'
+
+import type {State, Politician, PoliticianDetail, CadastralData, AssetDeclaration} from '../state'
+import type {Selector} from 'reselect'
```

Comment:
Nice :+1: Didn't know about this. Btw. I have read an article about flowing the recompose HOC's:
https://github.com/acdlite/recompose/tree/master/types